compat/openswoole.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+use OpenSwoole\Util;
+
+// @see https://github.com/mezzio/mezzio-swoole/issues/110#issuecomment-1500174967
+// Override the swoole_set_process_name function
+if (version_compare(phpversion('openswoole'), '22.0.0', '>=')) {
```

Comment:
This should check if openswoole extension is installed first

Reply:
Actually, if openswoole users added this workaround to their code this will cause errors. So this should also be wrapped in function_exists.

Verification is needed that vendor autoload files are included after root composer.json autoloading rules

Reply:
> Actually, if openswoole users added this workaround to their code this will cause errors.

If there is a migration document I can document this. Ideally this package should take care of such differences and not the end user. I guess end users will be more than happy to have mezzio-swoole working out of the box without the need for them to provide custom code.